packages/tizen_app_manager/tizen/src/tizen_app_manager.h

```diff
@@ -16,18 +16,15 @@ class TizenAppManagerPlugin : public flutter::Plugin {
   using MethodResultPtr =
       std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>>;
   static void RegisterWithRegistrar(flutter::PluginRegistrar *registrar);
+  flutter::EncodableList &Applications();
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> launch_events_;
+  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>>
+      terminate_events_;
```

Comment:
Like `batter_plus` you can declare `flutter::EventSink<flutter::EncodableValue>` as `FlEventSink`.
Please refer here.

https://github.com/flutter-tizen/plugins/blob/2bed8cbf8c2e9914531ff8db354e050f37c260a4/packages/battery_plus/tizen/src/battery_plus_tizen_plugin.cc#L21-L27

Reply:
I applied it partially using the method you mentioned. Thank you.